coverage_point_calculator/src/speedtest.rs

```diff
@@ -0,0 +1,244 @@
+use chrono::{DateTime, Utc};
+use rust_decimal::Decimal;
+use rust_decimal_macros::dec;
+
+const MIN_REQUIRED_SPEEDTEST_SAMPLES: usize = 2;
+const MAX_ALLOWED_SPEEDTEST_SAMPLES: usize = 6;
+
+#[derive(Debug, Default, Clone, Copy, PartialEq, PartialOrd)]
+pub struct BytesPs(u64);
+
+impl BytesPs {
+    pub fn new(bytes_per_second: u64) -> Self {
+        Self(bytes_per_second)
+    }
+
+    pub fn mbps(megabytes_per_second: u64) -> Self {
+        Self(megabytes_per_second * 12500)
+    }
+
+    fn as_mbps(&self) -> u64 {
+        self.0 / 12500
+    }
+}
+
+#[derive(Debug, Default, Clone, Copy, PartialEq)]
+pub struct Millis(u32);
```

Comment:
this should just be a type def

Reply:
`Millis` along with `Meters` have been demoted to type aliases.